src/components/loginForm/LoginForm.js

```diff
@@ -1,9 +1,27 @@
 import React from 'react';
 
-export const LoginForm = () => {
+export const LoginForm = ({
+  setUsername,
+  setPassword,
+  submit}) => {
   return (
-    <div style={{border: "1px solid black"}}>
-      <h3>Login Form Component</h3>
+    <div>
+      Log In<br/>
+      <input 
+        type="text" 
+        id="username" 
+        placeholder="Username"
+        onChange={e => setUsername(e.target.value)}
+      /><br/>
+      
+      <input 
+        type="password" 
+        id="password" 
+        placeholder="Password"
+        onChange={e => setPassword(e.target.value)}
+      /><br/>
+      
+      <button onClick={() => submit()}>Log In</button>
```

Comment:
This will login even if the username and password are empty. Want to add that check?